packages/dashboard/tests/frontend_selenium/pages/dashboard.py

```diff
@@ -4,84 +4,110 @@
+    login_button = (By.XPATH, '/html/body/div[2]/div[27]/div[2]/div[2]/div[2]/v-tab-item[1]/div/form/div[3]/button[2]')
+    login_password_input = (By.XPATH, '/html/body/div[2]/div[27]/div[2]/div[2]/div[2]/v-tab-item[1]/div/form/div[2]/div/div[1]/div/div[3]/input')
+    accept_terms_condition_button = (By.XPATH, '/html/body/div[2]/div[28]/div[2]/button')
+    connect_manual_button = (By.XPATH, '/html/body/div[2]/div[27]/div[2]/div[2]/div[2]/div[2]/div[2]/section/p/a')
+    connect_google_button = (By.XPATH, '/html/body/div[2]/div[27]/div[2]/div[2]/div[2]/div[2]/div[2]/section/div/a[1]')
+    connect_apple_button = (By.XPATH, '/html/body/div[2]/div[27]/div[2]/div[2]/div[2]/div[2]/div[2]/section/div/a[2]')
     terms_iframe = (By.TAG_NAME, 'iframe')
     iframe_load_image = (By.XPATH, '//*[@id="main"]/p[1]/img')
     tf_iframe_button = (By.XPATH, '/html/body/main/aside/div[2]/ul[3]/li/a')
     iframe_dialog_button = (By.XPATH, '//*[@id="cc_dialog"]/div/div[2]/button[1]')
-    accept_terms_condition_button = (By.XPATH, '//*[@id="app"]/div[4]/div/button')
-    mnemonic_input_reveal_button = (By.XPATH, '//*[@id="app"]/div[3]/div/div/div[3]/div/form/div/div/div[1]/div[2]/div/button')
-    mnemonic_login_input = (By.XPATH, '/html/body/div[1]/div[3]/div/div/div[3]/div/form/div/div/div[1]/div[1]/input')
-    logout_button = (By.XPATH, "//*[contains(text(), 'Logout')]")
-    tft_price_label = (By.XPATH, '//*[@id="app"]/div[1]/div[1]/header/div/div[2]/div/div/div/p[2]')
-    tft_swap_button = (By.XPATH, '//*[@id="app"]/div[1]/div[1]/header/div/div[2]/div/div/div/button/span/i')
-    tft_info_button = (By.XPATH, '//*[@id="app"]/div[1]/div[1]/header/div/div[2]/div/div/div/button[2]/span/i')
-    stellar_tft_price_label = (By.XPATH, '/html/body/div/div/div[3]/div/div/div[3]/div[1]/div[1]/dl/dd[9]/span')
-    manual_button = (By.XPATH, "//*[contains(text(), 'the manual')]")
-    connect_manual_button = (By.XPATH, "//*[contains(text(), 'ThreeFold Connect')]")
-    connect_google_button = (By.XPATH, '//*[@id="app"]/div[3]/div/div/div[3]/div/section/div/a[1]')
-    connect_apple_button = (By.XPATH, '//*[@id="app"]/div[3]/div/div/div[3]/div/section/div/a[2]')
+    mnemonic_input_reveal_button = (By.XPATH, '/html/body/div[2]/div[27]/div[2]/div[2]/div[2]/div[2]/div[1]/div[1]/div[1]/div/div[4]/i')
+    mnemonic_login_label = (By.XPATH, '/html/body/div[2]/div[27]/div[2]/div[2]/div[2]/div[2]/div[1]/div[1]/div[1]/div/div[3]/input')
+    key_type = (By.XPATH, '/html/body/div[2]/div[27]/div[2]/div[2]/div[2]/v-tab-item/div/form/div[2]/div[2]/div/div[1]/div/div[4]/i')
 
 
     def __init__(self, browser):
         self.browser = browser
     
     def open_and_load(self):
         self.browser.get(Base.base_url)
-        WebDriverWait(self.browser, 30).until(EC.visibility_of_element_located(self.dashboard_load_label))
+        WebDriverWait(self.browser, 30).until(EC.visibility_of_element_located(self.profile_load_label))
+        time.sleep(3)
```

Comment:
I added this only sleep here because of the new loading screen, which is very hard to wait on because most of the time it doesn't show up, so the wait does take more than 30 seconds just to fail, and if it appears, I have to wait another 30 seconds for it to disappear.

So 3 seconds seems to be more efficient, but if you still have any other idea, I could apply it better than sleep and wait. Please advise me with it, @mohamedamer453.